src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface OperationHandler {
```

Comment:
BTW, forgot about that in the first look, we also need to cover handlers with tests, and probably even the strategy class that provides handlers